cmd/pebble/cmd_enter.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/canonical/pebble/internal/logger"
+	"github.com/jessevdk/go-flags"
```

Comment:
This needs to be changed to Canonical's version of go-flags instead of just using go.mod. There's a long term commitment here as we have no indication of upstream's intention to address the changes required, and it shoud be clear to people reading these files that they are indeed looking at something else.

Reply:
Please note this is true of all the files under cmd/, not just this one.

Reply:
Mentioned this in more detail out of band. Module name of our fork has to be changed first: https://github.com/canonical/go-flags/pull/3#pullrequestreview-1368307459

I will update all the imports after the forked module name change.